<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageWidget</class>
 <widget class="QWidget" name="ManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ManageWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="ScreenLoginWidget" name="Login_page"/>
     <widget class="ConfigIPIDWidget" name="Config_page"/>
     <widget class="Start_InitData" name="Start_InitData_page"/>
     <widget class="MainWidget" name="MainWidget_page"/>
     <widget class="MainWidget" name="ManageWidget_page"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Start_InitData</class>
   <extends>QWidget</extends>
   <header>Start_InitData.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainWidget</class>
   <extends>QWidget</extends>
   <header>MainWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigIPIDWidget</class>
   <extends>QWidget</extends>
   <header>ConfigIPIDWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScreenLoginWidget</class>
   <extends>QWidget</extends>
   <header>ScreenLoginWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
